unlock-app/src/components/content/event/Registration/WalletlessRegistration.tsx

```diff
@@ -373,6 +373,24 @@ export const RegistrationForm = ({
         </>
       )}
 
+      <Controller
```

Comment:
Move to the top so that it does not look too awkward when there is an extra checkbox!